shared/src/commonMain/kotlin/com/karumi/gallery/data/local/LocalPhotosDataSource.kt

```diff
@@ -0,0 +1,35 @@
+package com.karumi.gallery.data.local
+
+import com.karumi.gallery.data.local.LocalGalleryDb.photosQueries
+import com.karumi.gallery.domain.TTLCache
+import com.karumi.gallery.model.PhotoShot
+import com.karumi.gallery.model.Photos
+
+class LocalPhotosDataSource(private val cache: TTLCache) {
+
+  fun insert(photos: Photos) {
+    photos.map(::insert)
+  }
+
+  private fun insert(photos: PhotoShot) {
+    photosQueries.transaction {
+      photosQueries.insertPhoto(
+        photos.id,
+        photos.thumbnailUrl,
+        photos.authorName,
+        photos.numberOfLikes
+      )
+    }
+    cache.persisTime()
```

Comment:
```suggestion
    cache.persistTime()
```

PD: Don't apply the suggestion from Github, do it in your IDE or it will break the project

Reply:
thanks!